configure.ac

```diff
@@ -369,12 +369,18 @@ fi
 
 if test -n "$with_portals4" -a "$with_portals4" != "no" ; then
     transport="portals4"
+    transport_ofi="no"
+    transport_ucx="no"
     AC_DEFINE([USE_PORTALS4], [1], [Define if Portals 4 transport active])
 elif test -n "$with_ofi" -a "$with_ofi" != "no" ; then
     transport="ofi"
+    transport_portals="no"
+    transport_ucx="no"
     AC_DEFINE([USE_OFI], [1], [Define if OFI transport active])
 elif test -n "$with_ucx" -a "$with_ucx" != "no" ; then
     transport="ucx"
+    transport_ofi="no"
+    transport_portals="no"
     AC_DEFINE([USE_UCX], [1], [Define if UCX transport active])
```

Comment:
Looks good to me. Just a thought, it seems we can merge this if-else block to the multiple if statements above. We are any way error'ing out when multiple transports are selected using `num_transport`.

Reply:
@wrrobin I did the merge you suggested, and I think it's clearer this way.

@jdinan @rdesai16 - might want to do a quick re-review if possible to make sure we didn't make a mistake.